lib/matplotlib/backend_bases.py

```diff
@@ -3264,15 +3266,16 @@ def draw_if_interactive(cls):
 
     @classmethod
     @cbook._make_keyword_only("3.1", "block")
-    def show(cls, block=None):
+    def show(cls, block=None, *, figures=None):
```

Comment:
```suggestion
    def show(cls, figures, *, block=None):
```
because this PR has stalled enough so that `block` has become kw-only meanwhile :smirk:.

Reply:
yup